libs/books/feature/src/lib/reading-list/reading-list.component.ts

```diff
@@ -15,4 +20,16 @@ export class ReadingListComponent {
   removeFromReadingList(item) {
     this.store.dispatch(removeFromReadingList({ item }));
   }
-}
+
+  markBookAsFinished(item: ReadingListItem): void {
+    this.store.dispatch(
+      markBookAsFinished({
+        item: {
+          ...item,
+          finished: true,
```

Comment:
You are not using `finished` and `finishedDate` value from here, inside service file.
You can remove it.